components/SearchResultsList.vue

```diff
@@ -38,13 +38,12 @@
                 </div>
             </div>
         </div>
-
         <div v-else>
-            <div class="h-full flex flex-col">
-                <SVGNoSearchResults class="p-12" />
-                <span class="text-primary-text pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">{{
+            <div class="flex flex-col">
+                <SVGNoSearchResults class="portrait:px-4 h-12" />
```

Comment:
Design nit: personally I think the svg is a little too small now on both landscape and portrait mode. Could we make it a little bigger? (Doesn't have to be as big as the original)

Reply:
Looking on my phone, I think about double looks right. (Originally was like 4x)

Reply:
@ermish If we make it bigger on landscape, the "No Results Found" text won't be visible, which I think is more important for the user to see. We could consider moving the text above the SVG.

Reply:
Valid point. Let's swap it whenever you have time